SUI/SUI/ContentView.swift

```diff
@@ -8,17 +8,113 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let firstText = "Пример Alert"
+        static let secondTestText = "foo"
+        static let secondText = "Пример Alert 2 с 2 кнопками и логикой"
+        static let firstAlertName = "Показать"
+        static let closeButtonText = "Close"
+        static let secondAlertName = "Показать"
+        static let secondAlertTitle = "Wanna change label?"
+        static let secondAlertPrimaryButtonText = "Change label"
+        static let firstActionSheetText = "Пример Action Sheet"
+        static let secondActionSheetText = "Пример Action Sheet с кнопками и логикой"
+        static let firstActionSheetName = "Показать"
+        static let actionSheetConfDialogText = "Are you sure?"
+        static let testButtonText = "test"
+        static let testPrint = "its worked"
+        static let secondActionSheetName = "Показать"
+        static let secondActionSheetButtonName = "Change color"
+    }
+    
+    @State var isFirstAlertShown = false
+    @State var isSecondAlertShown = false
+    @State var isActionButtonTapped = false
+    @State var isFirstActionSheetShown = false
+    @State var isSecondActionSheetShown = false
+    @State var isSecondActionSheetButtonTapped = false
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: 15) {
+            Text(Constants.firstText)
+            showFirstAlert()
+            if !isActionButtonTapped { Text(Constants.secondText) } else {
+                Text(Constants.secondTestText)
+            }
+            showSecondAlert()
+            Text(Constants.firstActionSheetText)
+            showFirstActionSheet()
+            Text(Constants.secondActionSheetText)
+            showSecondActionSheet()
+                .padding()
+        }
+    }
+    
+    fileprivate func showFirstAlert() -> some View {
+        return Button {
+            self.isFirstAlertShown = true
+        } label: {
+            Text(Constants.firstAlertName)
+        }.alert(isPresented: $isFirstAlertShown) {
+            Alert(title: Text(Constants.closeButtonText))
+        }
+    }
```

Comment:
```suggestion
    var showFirstAlert: some View {
        return Button {
            self.isFirstAlertShown = true
        } label: {
            Text(Constants.firstAlertName)
        }.alert(isPresented: $isFirstAlertShown) {
            Alert(title: Text(Constants.closeButtonText))
        }
    }
```
для выноса логики которая ничего не принимает, а возвращает some View используем вычисляемые свойства